packages/ui/components/image-uploader/BannerUploader.tsx

```diff
@@ -224,5 +230,6 @@ async function getCroppedImg(
     canvas.height
   );
 
-  return canvas.toDataURL("image/png");
+  // Use original format with quality setting for JPEG
+  return canvas.toDataURL(originalFormat, originalFormat === "image/jpeg" ? 0.6 : undefined);
```

Comment:
We don't force convert to png, which was the primary cause for the crazy size jump.